### ISSUE TITLE
send empty string if no values are set - feature request

### ISSUE THREAD

**Issue by papppeter** [unclead/yii2-multiple-input#307]
HI,

it would be nice if it could work like checkboxes. If nothing is there we still send something to post (empty string) to be able to determine it was on that form.

This feature request would be essential to this extension:
la-haute-societe/yii2-save-relations-behavior/issues

example:
2 elements in multi input:
_POST[
 model[
  [..]
  [..]
 ]
]
0 element in multi input
_POST[
 model =>''
]

than the realationsave behavior could determine to delete all the related instances.

Quick fix is:
```
<?= \yii\helpers\Html::hiddenInput('model '); ?>
<?= TabularInput::widget([ ...
```

**Comment by papppeter** [comment]
related issue: [#252](https://github.com/unclead/yii2-multiple-input/issues/252)

**Comment by unclead** [comment]
closed as a duplicate of #252
